src/test/java/com/zebrunner/carina/demo/WebSampleTest.java

```diff
@@ -143,4 +147,56 @@ public void testBrandGroup() {
         softAssert.assertAll();
     }
 
+    @Test(description = "Validating search input field redirection")
+    @MethodOwner(owner = "emir")
+    public void searchInputTest(){
+        //   //a[contains(text(),'A')] XPath para buscar elementos que contengan dentro de su texto, dicho texto asignado
+        //   //a[1] para buscar en base a los elementos hijos de un elemento padre el enesimo elemento hijo que tiene
+        //Tambien se puede consultar la XPath desde el inspector con click derecho y copy XPath
+
+        MyHomePage homePage = new MyHomePage(getDriver());
+
+        WebDriver driver = getDriver();
+//        driver.navigate().to("https://www.gsmarena.com");
+        homePage.openPage();
+
+        //Va a la pagina de celulares de Nokia
+        //driver.findElement(By.xpath("//*[@id=\"body\"]/aside/div[1]/ul/li[4]/a")).click();
+        //driver.findElement(By.xpath("//a[text()=\"Nokia\"]"));
+
+//        driver.findElement(By.cssSelector("#topsearch>input")).sendKeys("Huawei");
+//        driver.findElement(By.cssSelector("#topsearch>input")).sendKeys(Keys.ENTER);
+        homePage.searchInput("Huawei");
+
+        String searchText = driver.findElement(By.tagName("h1")).getText();
```

Comment:
It is not correct to work with locors in test methods

Reply:
Oh this was the test from the theoretical class hehe, was an error add it to the PR